src/main/java/boost/project/PackageSpringBootAppWithLibertyMojo.java

```diff
@@ -223,4 +253,19 @@ private void createUberJar() throws MojoExecutionException {
     		)
 		);
 	}
+
+	/**
+	* Detect spring boot version dependency
+	*/
+	private String findSpringBootVersion(MavenProject project) {
+        	String version = null;
+        	List<Dependency> dependencies = project.getDependencies();
+        	for(Dependency dep: dependencies) {
+            		if("org.springframework.boot".equals(dep.getGroupId())){
```

Comment:
Would it be better to narrow this down to the "spring-boot-starter-parent" artifact specifically within the group?   Not sure if there's a valid case where the parent starter comes from one version but some other dependency in the group comes from another (1.5.x vs. 2.0.x).

Reply:
This is how Anjum recommended checking the version of the spring boot dependency. The dependency versions end up coming from the parent pom (several levels of indirection up). Plus, the application could reference a different parent pom that then lists the spring boot one as its parent. In which case, the version would not be the one we are looking for.

Reply:
Let's talk more tomorrow about this mix version case and whether it is valid for Open Liberty.